### PR TITLE
Meenu/chore: added search under block

### PR DESCRIPTION
**Pull request by meenakshi-deriv**

### CODE REVIEW

**Comment by mohsen-deriv** [comment]
@meenakshi-deriv is this ready?

**Review by mohsen-deriv — CHANGES_REQUESTED** [review]
please resolve conflicts

File: libs/blocks/index.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from '../../dist/libs/blocks/deriv-com-v2/libs/blocks/src/index'
```
Comment (mohsen-deriv) [inline libs/blocks/index.d.ts:1]: why do we need this?

**Review by mohsen-deriv — CHANGES_REQUESTED** [review]

File: libs/blocks/index.d.ts (hunk 1 above)
Comment (meenakshi-deriv) [reply libs/blocks/index.d.ts:1]: strang. Not my changes. Let me check.

File: package.json [hunk 2]
```diff
@@ -30,6 +30,7 @@
     "next": "13.4.19",
     "next-image-export-optimizer": "^1.8.5",
     "react": "18.2.0",
+    "react-device-detect": "^2.2.3",
```
Comment (prince-deriv) [inline package.json:33]: Let's remove this. It's not being used anymore.

File: libs/blocks/src/lib/search/index.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { SearchChip } from '@deriv-com/components';
+import { ReactNode } from 'react';
+
+export type SearchBlockProps = {
+  showSearchresults: (e: string) => void;
+  content?: ReactNode;
+  searchPlaceholder: string;
```
Comment (prince-deriv) [inline libs/blocks/src/lib/search/index.tsx:7]: `placeholder` alone I think is enough 
```suggestion
  placeholder: string;
```

**Review by prince-deriv — CHANGES_REQUESTED** [review]

**Pull request merged**